src/matlab2tikz.m

```diff
@@ -3424,6 +3424,16 @@ case size(cData,2) % specific y-data is given
     end
 end
 % ==============================================================================
+function BarWidth=getCorrectBarWidth(h)
```

Comment:
The *Correct* part is not informative in the name of the function. I suggest to either drop it or add a more specific adjective but such that the name starts with `getBar...`